### ISSUE TITLE
Backup and Restore settings

### ISSUE THREAD

**Issue by jackuars89** [8bitPit/Niagara-Issues#1939]
### Is your feature request related to a problem?

This feature is absent in Niagra and is a pain while uninstalling it and reinstalling it when trying other launchers.

### Describe the solution you'd like

Have a backup and restore setting

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by Oddward** [comment]
This is a duplicate of https://github.com/8bitPit/Niagara-Issues/issues/222 (with another duplicate made the same day 😅), and there's more info in the knowledge base article here: [Backup settings](https://help.niagaralauncher.app/article/37-backup-settings)

Tl;dr: It currently already uses a Google auto backup solution, but it's a hit or miss, so a different method is planned.

If you're reinstalling on the same device it should have a local backup as long as you don't opt to remove all app data when you uninstall it (a checkbox pops up).
